demo/src/plugins.rs

```diff
@@ -102,8 +102,11 @@ impl ClickWatcher {
 
 impl Plugin for &mut ClickWatcher {
     fn run(&mut self, response: &Response, painter: Painter, projector: &Projector) {
-        if !response.changed() && response.clicked_by(egui::PointerButton::Primary) {
-            if let Some(offset) = response.hover_pos().map(|p| p - response.rect.center()) {
+        if response.clicked_by(egui::PointerButton::Primary) {
+            if let Some(offset) = response
+                .interact_pointer_pos()
+                .map(|p| p - response.rect.center())
+            {
                 self.clicked_at = Some(projector.unproject(offset));
```

Comment:
Now when I look at it, `unproject` could be moved to `.map`, i.e. something like:

```rust
self.clicked_at = response
                .interact_pointer_pos()
                .map(|p| projector.unproject(p - response.rect.center()));
```

That's one less `if`.

Reply:
actually, `!response.changed() &&` lost accidentally